### PR TITLE
Release 3.4.1

### PR DESCRIPTION
**Pull request by FinalAngel**
- added ``button-variant-outline``mixin
- added ``.spacer-sm`` an ``spacer-md``
- changed line length rule for JS & HTML
- changed ``.jshintrc`` settings slightly
- improved ``.travis.yml`` structure

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
 
 node_js:
-  - "0.10"
+  - 0.10
+
+sudo: false
+
+env:
+  global:
+    - secure: ghtK6PdDy13ruL+HGT4doVcUkl8pyzukY1gkHbkIAL8BrOc0eQdGBnAxKpPGZJv/8upWyul69DgXKg26cbS633Gfp0NjM/YPUh6wEgTg46Zjirw1ejX7xmqsQUGukZXOXNsll7/Syql+KEn8ao+JOAh+gjkFfNY2mxD6Oxs56Sc=
```
Comment (vxsx) [inline .travis.yml]: comments missing

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "private": true,
     "devDependencies": {
         "browser-sync": "2.7.12",
-        "browserslist-saucelabs": "0.1.2",
+        "browserslist-saucelabs": "^0.1.2",
```
Comment (vxsx) [inline package.json]: might as well change it to 0.1.4

File: private/sass/layout/_buttons.scss [hunk 3]
```diff
@@ -13,58 +13,23 @@
...
+
     &.btn-default {
-        @include button-variant($btn-default-color, transparent, $btn-default-border);
-        &:hover,
-        &:active,
-        &:focus {
-            color: #fff;
-            background-color: $btn-default-border;
-        }
+        @include button-variant-outline($btn-default-color, $btn-default-border);
```
Comment (vxsx) [inline private/sass/layout/_buttons.scss]: <3

File: .travis.yml (hunk 1 above)
Comment (FinalAngel) [inline .travis.yml]: that's done in https://github.com/aldryn/aldryn-boilerplate-bootstrap3/blob/feature/4.0.0/.travis.yml, will add on 4.x release

**Comment by Brankub** [comment]
LGTM

**Comment by vxsx** [comment]
LGTM

**Pull request merged**